lib/schedule.c

```diff
@@ -149,17 +149,15 @@ trigger_handle_t lf_schedule_trigger(environment_t* env, trigger_t* trigger, int
   if (!trigger->is_timer) {
     delay += trigger->offset;
   }
-  tag_t intended_tag = (tag_t){.time = env->current_tag.time + delay, .microstep = 0};
+  tag_t intended_tag = lf_delay_tag(env->current_tag, delay);
+  printf("intended tag = " PRINTF_TAG ".\n", intended_tag.time, intended_tag.microstep);
```

Comment:
This looks like a debug statement. It should be deleted or replaced with `LF_PRINT_DEBUG`.

Reply:
Oh, I made a mistake here! Thank you so much for capturing it. Resolved in [5c966b5](https://github.com/lf-lang/reactor-c/pull/390/commits/5c966b50a131053c738a93ffd7329b020277aa49).